x-pack/plugins/apm/public/components/shared/Stacktrace/index.tsx

```diff
@@ -113,7 +139,19 @@ class Stacktrace extends PureComponent {
   }
 }
 
-function Libraryframes({ visible, stackframes, codeLanguage, onClick }) {
+interface LibraryframesProps {
+  visible?: boolean;
+  stackframes: Stackframe[];
+  codeLanguage?: string;
+  onClick: () => void;
+}
+
+const Libraryframes: React.SFC<LibraryframesProps> = ({
```

Comment:
Can you move this to its own file? 
Generally I like the "one component per file" - unless the components are very simple/small.